sdk/tables/azure-data-tables/README.md

```diff
@@ -88,11 +92,12 @@ The `credential` parameter may be provided in a number of different forms, depen
     table_service_client = TableServiceClient(account_url="https://<my_account_name>.table.core.windows.net", credential=sas_token)
 ```
 
-2. To use an account [shared key](https://docs.microsoft.com/rest/api/storageservices/authenticate-with-shared-key/)
+2. To use an account [shared key][azure_shared_key]
    (aka account key or access key), provide the key as a string. This can be found in the Azure Portal under the "Access Keys"
    section or by running the following Azure CLI command:
 
-    ```az storage account keys list -g MyResourceGroup -n mystorageaccount```
+    ```az storage account keys list -g MyResourceGroup -n MyStorageAccount```
+
     Use the key as the credential parameter to authenticate the client:
     ```python
     from azure.data.tables import TableServiceClient
```

Comment:
Is it possible to organize these two sections (#### Creating the client from a connection string and #### Create the client) so that it's clearer that they are alternatives? maybe having create a client as `###` and then as `####` `From a Connection String` and `whatever is a good description of the other one` and then all of the sections that fall underneath each of them even one level deeper (H5)